test-globals.ts

```diff
@@ -0,0 +1,3 @@
+export const setup = () => {
+  process.env.TZ = 'Europe/Moscow';
```

Comment:
Please set to some other timezone so that we can see it being reflected in tests for real.